go-controller/pkg/node/gateway_shared_intf_linux.go

```diff
@@ -39,12 +39,7 @@ func setupLocalNodeAccessBridge(nodeName string, subnets []*net.IPNet) error {
 		return err
 	}
 
-	var macAddress string
-	if config.IPv4Mode {
-		macAddress = util.IPAddrToHWAddr(net.ParseIP(types.V4NodeLocalNATSubnetNextHop)).String()
-	} else {
-		macAddress = util.IPAddrToHWAddr(net.ParseIP(types.V6NodeLocalNATSubnetNextHop)).String()
-	}
+	macAddress := util.IPAddrToHWAddr(net.ParseIP(types.V4NodeLocalNATSubnetNextHop)).String()
```

Comment:
@trozet should we have a comment about this in the code rather than just git history? eg that this MAC is always generated from the v4 address.

Reply:
I think it's OK without because we are going to deprecate all this code when we get rid of DGP soon